gulpfile.js

```diff
@@ -150,6 +235,48 @@ const nw = new nwBuilder({
 /*************
  * gulp tasks *
  *************/
+gulp.task('clean-js', (done) => {
+  del(['src/app/js/*.js'], {force: true}).then(done());
```

Comment:
`.then(done)` or `.then(() => done());`. this calls done immediately

Reply:
Indeed, my bad for missing that.